src/test/java/winstone/accesslog/SimpleAccessLoggerTest.java

```diff
@@ -40,6 +40,7 @@ public void testSimpleConnection() throws Exception {
         makeRequest("http://localhost:"+port+"/examples/CountRequestsServlet");
 
         // check the log file
+        Thread.sleep(5000);
```

Comment:
would be better to add a loop retrying every maybe 100ms then if keep failing after 5000ms
because this add unconditionally 5s to the build

Reply:
I am not interested in doing that.

Reply:
recorded in https://github.com/jenkinsci/winstone/issues/231 for future improvement